src/Features/CSharpTest/IntroduceUsingStatement/IntroduceUsingStatementTests.cs

```diff
@@ -1417,4 +1421,40 @@ void M(System.IDisposable disposable)
             }
             """, LanguageVersion.CSharp8);
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/37260")]
+    public async Task TestExpressionStatement()
+    {
+        await TestAsync(
+            """
+            using System;
+
+            class C
+            {
+                void M()
+                {
+                    [||]MethodThatReturnsDisposableThing();
+                    Console.WriteLine();
+                }
+
+                IDisposable MethodThatReturnsDisposableThing() => null;
+            }
+            """,
+            """
+            using System;
+            
+            class C
+            {
+                void M()
+                {
+                    using (MethodThatReturnsDisposableThing())
```

Comment:
>using (MethodThatReturnsDisposableThing()) [](http://example.com/codeflow?start=20&length=42)

Is there an option for indicating the using form desired?

using (...) {}

vs

using var _ = ... ;

Reply:
done